parsl/providers/local/local.py

```diff
@@ -230,8 +228,9 @@ def submit(self, command, tasks_per_node, job_name="parsl.localprovider"):
                                   stdout, stderr)
         for line in stdout.split('\n'):
             if line.startswith("PID:"):
-                remote_pid = line.split("PID:")[1].strip()
+                remote_pid = int(line.split("PID:")[1].strip())
                 job_id = remote_pid
```

Comment:
Where does this get used?  I didn't search all the places but the docstring specifies job_id is a string, but you've changed it to int.  I assume the int() is because os.kill expects an int.

Reply:
That's a good point.  Yes, `remote_pid` should be an integer given `os.kill()`, but the external interface of `job_id` should be a string.  Thanks.